apps/angularlogos/src/app/logos/logos.component.html

```diff
@@ -15,20 +15,40 @@ <h1>
     Note: All logos on this site are the property of their respective owners.
   </p>
   <hr />
-  <p>
-    <strong>Want to add your logo?</strong><br />
-    Create a Pull Request on our GitHub repo or file an issue.
-  </p>
-  <a
-    class="contribute"
-    mat-raised-button
-    color="primary"
-    href="https://github.com/maartentibau/angular-logos/pulls"
-    target="_blank"
-    role="button"
-    rel="noreferrer"
-    >Add your logo</a
-  >
+  <div class="actions">
+    <div>
+      <p>
+        <strong>Want to add your logo?</strong><br />
+        Create a Pull Request on our GitHub repo or file an issue.
+      </p>
+      <a
+        class="contribute"
+        mat-raised-button
+        color="primary"
+        href="https://github.com/maartentibau/angular-logos/pulls"
+        target="_blank"
+        role="button"
+        rel="noreferrer"
+        >Add your logo</a
+      >
+    </div>
+    <div>
+      <p>
+        <strong>No Logo right now?</strong><br />
```

Comment:
People visiting this website probably don't want to submit or create logos but just see them. I think that just a minority want to create logos.
Suggestion:

```suggestion
        <strong>You want to create a logo?</strong><br />
```

Reply:
However, I see your intention to input it next to the "submit your logo" action. This would also be okay for me:

```suggestion
        <strong>No Logo yet?</strong><br />
```

Reply:
cc @maartentibau

Reply:
both fine for me :)

Reply:
Both fine for me as well